pkgs/top-level/all-packages.nix

```diff
@@ -17585,6 +17585,18 @@ in
   emacsPackages = emacs26Packages;
   emacsPackagesNg = emacs26PackagesNg;
 
+  emacsHead = callPackage ../applications/editors/emacs/head.nix {
+    # use override to enable additional features
+    libXaw = xorg.libXaw;
+    Xaw3d = null;
+    gconf = null;
+    alsaLib = null;
+    imagemagick = null;
```

Comment:
We don't support ImageMagick?

Reply:
in emacs 27 it's disabled by default, recent replaced by something else. I haven't actualy c hanged these options from what emacs26 called default.nix with anyways